bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/BootstrapSettings.java

```diff
@@ -53,6 +57,7 @@ public BootstrapSettings() {
         this.updateSecurityManager = true;
         this.autoAppendResources = true;
         this.useCdnResources = false;
+        this.useBundle = false;
```

Comment:
Maybe we should switch to `true` as default. Otherwise your reported [issue](https://github.com/l0rdn1kk0n/wicket-bootstrap/issues/951) would be still valid, no ?

Reply:
Well,
popper version need to be fixed, via (additional) commit

IMO it worth to switch to bundle-only mode, this would make things simpler :)

Reply:
I agree with bundle-only setup!

Reply:
Done :)

`MaterialDesignTheme` and `ItaliaTheme` still requires Popper (seems to)
but they are for bootstrap4 only :(

Shall we keep or drop it?

Reply:
I'd suggest to just remove these lines:
* https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/7010be70febdbe34d8adad3629b501056f767a8b/bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/italia/ItaliaTheme.java#L54
* https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/7010be70febdbe34d8adad3629b501056f767a8b/bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/material_design/MaterialDesignTheme.java#L32

Reply:
done
Also dropped useless dependency :)

Reply:
https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/7010be70febdbe34d8adad3629b501056f767a8b/bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/material_design/MaterialDesignTheme.java#L32
Do we still need this ?

Reply:
this one is dropped already
will lead to compilation error otherwise :)

Reply:
Right! I am looking at the target branch ... 🤦🏻